src/app/shared/form/button/button.component.ts

```diff
@@ -0,0 +1,49 @@
+import { NgModule, Component, OnInit, Input, ViewChild, ElementRef, Renderer2 } from '@angular/core';
+
+@Component({
+  selector: 'in-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+  @Input() fontSize: string;
+  @Input() content: string;
+  @Input() fontColor: string;
+  @Input() border: string;
+  @Input() borderRadius: string;
+  @Input() borderColor: string;
+  @Input() backgroundColor: string;
+  @Input() width: number;
+  @Input() height: number;
+  @Input() textAlign: string;
+  @Input() loading: boolean;
+  @ViewChild('button') button: ElementRef;
+
+  constructor(private renderer: Renderer2) { }
+
+  ngOnInit() {
+    this.border = this.border ? this.border : 'none';
```

Comment:
default values for input should be placed after `@Input()` declaration i.e. `@Input() borderColor = 'CornflowerBlue'`, also type declaration could be omitted because typescript implicitly set this to string.
Tip - ternary operator is not necessary here, this expression could be written like this: 
`this.border = this.border || 'none';`